editor/src/clj/editor/settings_core.clj

```diff
@@ -38,22 +55,26 @@
   (BufferedReader. (StringReader. content)))
 
 (defn- empty-parse-state []
-  {:current-category nil :settings nil})
+  {:current-category nil :settings []})
```

Comment:
We need to start with an empty vector instead of nil. The 'conj' operator behaves differently depending on if it is nil or an empty vector. If it you conj nil with something you get a List, but we really want a Vector to be able to use assoc-in from set-setting.